js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -18,6 +18,7 @@ import { Contact } from './types/contact';
 import { MessageToContact } from './types/message-to-contact';
 import { Points } from './types/points';
 import { HostEnvironmentInfo } from './types/host-environment-info';
+import { DownloadHeaders } from './types/download-headers';
```

Comment:
May be DownloadFileHeaders 🤔

Reply:
@rleojoseph do you mean a rename for the type overall?